frontend/common/src/lang/fr.json

```
@@ -1795,5 +1795,149 @@
+    "description": "Text that appears when a user is missing optional skills on their profile"
+  },
+  "VqrVpT": {
+    "defaultMessage": "Mise à jour impossible – cette adresse courriel est déjà utilisée.",
+    "description": "Error message that the given email address is already in use when updating."
+  },
+  "WUAAr1": {
+    "defaultMessage": "Création impossible – cette adresse courriel est déjà utilisée.",
+    "description": "Error message that the given user identifier is already in use when creating."
+  },
+  "Wwb8Lb": {
+    "defaultMessage": "Sélecteur de mode de couleur du thème",
+    "description": "Label for the group of buttons to change the current colour mode"
+  },
+  "XNDPQM": {
+    "defaultMessage": "Un champ obligatoire est vide : Date de fin",
+    "description": "Error message that the pool advertisement must have an expiry date."
+  },
+  "aMkZ80": {
+    "defaultMessage": "Vous devez indiquer le lieu précis en anglais et en français si l’annonce ne vise pas un poste à distance.",
+    "description": "Error message that advertisement locations must be filled in English and French."
+  },
+  "euwgms": {
+    "defaultMessage": "Un champ obligatoire est vide : Français – Votre travail",
+    "description": "Error message that Work Tasks in French must be filled"
+  },
+  "gU/2O6": {
+    "defaultMessage": "Le bassin doit avoir une date d’expiration postérieure à aujourd’hui.",
+    "description": "Error message that pool expiry isn't in the future."
+  },
+  "juklNA": {
+    "defaultMessage": "Un champ obligatoire est vide : Anglais – Votre incidence",
+    "description": "Error message that Your Impact in English must be filled"
+  },
+  "kg28xx": {
+    "defaultMessage": "Un champ obligatoire est vide : Français – Votre incidence",
+    "description": "Error message that Your Impact in French must be filled"
+  },
+  "l7Hif/": {
+    "defaultMessage": "Compétences sélectionnées",
+    "description": "Section header for a list of skills selected"
+  },
+  "mv7JO3": {
+    "defaultMessage": "Autre",
+    "description": "The publishing group called Other"
+  },
+  "nPKPFa": {
+    "defaultMessage": "Un champ obligatoire est vide : Groupe des publications",
+    "description": "Error message that the pool advertisement must have publishing group filled."
+  },
+  "r7x5H3": {
+    "defaultMessage": "S. O.",
+    "description": "Message displayed when specific information is not applicable"
+  },
+  "t4F/0R": {
+    "defaultMessage": "Un champ obligatoire est vide : Exigence relative à la sécurité",
+    "description": "Error message that the pool advertisement must have a security clearance."
+  },
+  "tW4k56": {
+    "defaultMessage": "Un champ obligatoire est vide : Anglais – Votre travail",
+    "description": "Error message that Work Tasks in English must be filled"
+  },
+  "w2tWfH": {
+    "defaultMessage": "Un champ obligatoire est manquant : Titres des volets/emplois",
```

For the sake of consistency.

```suggestion
    "defaultMessage": "Un champ obligatoire est vide : Titres des volets/emplois",
```